docs/review-tool/README.md

```diff
@@ -0,0 +1,243 @@
+# Overview
+
+Creating and publishing high-quality "smart" legal agreements from scratch is a though task,
+but now that you know more about OpenLaw Markup Language you can help us to automate this process.
+
+We’re converting tens of thousands of traditional “legacy” agreements, into “blockchain compatible” contracts -
+which are markdown files with our markup language.
+
+The goal is the build a huge library of "smart" legal agreements, so you can use any of these base documents to
+build your own version without converting a real world contract into markdown format.
+
+There are five main steps that compose this process to build up our library:
+
+1.  Collecting HTML agreements from various sources, such as [LawInsider.com](https://www.lawinsider.com/education);
+2.  Parsing and Converting HTML files into OpenLaw Markdown format using Natural Language Processing (NLP);
+3.  Uploading markdown files to the Review Tool;
+4.  Reviewing all the files to make sure they were properly converted;
+5.  Gathering review results to feed the NLP service and improve the conversion process.
+
+With these steps we have a feedback loop that leverages a new flow of information and raises the standards
+of the converted files.
+
+The Review Tool is an important piece of this puzzle and now our community can join forces with us and help
+to build up this library of agreements.
+
+# How-To
+
+## Sign Up
+
+- [Register](https://conversion.openlaw.io/register)
+
+## Sign In
+
+- [Login](https://conversion.openlaw.io/login)
+
+## Review
+
+### Modes
+
+The Review Tool allows you to whether reviewing a particular agreement -
```

Comment:
I think maybe something is missing from this sentence (decide whether to review a particular agreement or start from where you left off?)